mm2src/coins/eth.rs

```diff
@@ -2328,7 +2328,8 @@ impl MarketCoinOps for EthCoin {
                     Ok(conf) => {
                         if conf == confirmed_at {
                             status.append(" Confirmed.");
-                            break Ok(());
+                            // Let's avoid risking additional errors by calling `current_block` and instead return required_confirms
+                            break Ok(required_confirms.as_u64());
```

Comment:
Related to the above comment, If we are gonna return confirmations from `wait_for_confirmations` we should return the actual confirmations, otherwise it's not needed. Another dev can use this somewhere else and it just returns the confirmations passed to the function which is not accurate. I change my mind, let's not return number of confirmations from the `wait_for_confirmations` function :)

Reply:
> I change my mind, let's not return number of confirmations from the wait_for_confirmations function :)

Ive also started leaning towards this option, as it wasn't urgently needed, and some chain impls increase the risk of errors. Will revert it.

Reply:
Done https://github.com/KomodoPlatform/komodo-defi-framework/pull/2199/commits/0073fe80d5d89fbd055fa96c6199f73139b3dc9e